fixtures/docker.md

```diff
@@ -0,0 +1,16 @@
+# docker
```

Comment:
I see the man-page generated also includes `--help` as option, but don't see it in the Markdown variant here; is that expected, or is something missing?


It's possible that `--help` was an option that's added automatically (so not explicitly defined), so ... maybe related if that's the case?

Reply:
I wouldn't expect `--help` to show up as an option. It's an implicit flag in cobra, right? We don't define it as a flag anywhere so I'm not sure how the doc generation for it should work. I wonder how come it's being generated for the man page output, do you know @crazy-max ?

Reply:
Yeah, mostly noticed the difference between man-pages and YAML here. That said; the command-line will show `--help` as option, and given that these YAML-files are used to produce documentation, I somewhat would expect them to be complete;

```
      --help                             Print usage
```

I'm curious though of other flags _are_ included (see my comment below on the YAML file); is it because our root command didn't define any, or is something missing in the generator code to also include flags/options?

Reply:
I'm thinking since the purpose of this PR is to enable longdesc generation for the `docker` base command, I don't think this discrepance between `man` and other formats needs to block. We don't have `--help` for the base command today, so there's no difference really.